src/load_file.cpp

```diff
@@ -37,7 +37,11 @@ namespace miopen {
 
 std::vector<char> LoadFile(const fs::path& path)
 {
+#if MIOPEN_WORKAROUND_USE_BOOST_FILESYSTEM
```

Comment:
I would suggest declaring some alias via using in the fs header so that workaround macro check would not be propagated to other files.

Reply:
@DrizztDoUrden Thanks. Of course, I considered this, but decided to leave it as is because
- The alias would only be used once
- Introduces another level of indirection
- The diff would be one line longer